### ISSUE TITLE
collect organization info

### ISSUE THREAD

**Issue by 0xdabbad00** [duo-labs/cloudmapper#289]
Add to `collect_commands.yaml`
```
- Service: organizations
  Request: describe-organization
- Service: organizations
  Request: list-accounts
```

Edit `collect.py` to have `organizations` as gloabl (add other global services there as well).

Add check to ignore the error when you call list-accounts:
`An error occurred (AccessDeniedException) when calling the ListAccounts operation: You don't have permissions to access this resource.` 
That error happens unless you are on the organization master.

**Comment by 0xdabbad00** [comment]
While I'm in there, I should ignore the error when a password policy cannot be found.  These look like: `An error occurred (NoSuchEntity) when calling the GetAccountPasswordPolicy operation: The Password Policy with domain name XXX cannot be found.`

**Comment by 0xdabbad00** [comment]
Similar things to ignore:
```
ecr.get_repository_policy(XXX): An error occurred (RepositoryPolicyNotFoundException) when calling the GetRepositoryPolicy operation: Repository policy does not exist for the repository with name 'XXX' in the registry with id 'XXX'
kms.list_key_policies(XXX): An error occurred (AccessDeniedException) when calling the ListKeyPolicies operation: User: XXX is not authorized to perform: kms:ListKeyPolicies on resource: XXX
kms.get_key_rotation_status(XXX): An error occurred (AccessDeniedException) when calling the GetKeyRotationStatus operation: User: XXX is not authorized to perform: kms:GetKeyRotationStatus on resource: XXX
lambda.get_policy(XXX): An error occurred (ResourceNotFoundException) when calling the GetPolicy operation: The resource you requested does not exist.
  ```

**Comment by livehybrid** [comment]
Hi @0xdabbad00 👍  on the kms:ListKeyPolicies error

**Comment by 0xdabbad00** [comment]
Resolved in #349

I didn't filter out the KMS related issues though as I need to dig more into those as they are `AccessDeniedException` and not a resource not found.
